src/response/chi0.jl

```diff
@@ -110,6 +110,7 @@ function sternheimer_solver(Hk, ψk, ε, rhs;
                             callback=identity, cg_callback=identity,
                             ψk_extra=zeros_like(ψk, size(ψk, 1), 0), εk_extra=zeros(0),
                             Hψk_extra=zeros_like(ψk, size(ψk, 1), 0), tol=1e-9)
+    iszero(rhs) && return(zero(rhs))
```

Comment:
Also not sure. Comparisons against exact zero are always fishy, so it should probably be some tolerance. But which ?

Also don't braket around the `return` ;)

Reply:
`Tolerance of Sterneimer/10` is tricky, because what is a good cutoff depends on the conditioning of `H-ε`. I mean what you do now fixes the immediate issue, but it does not guarantee we don't fall into this trap again when the `rhs` is small.

But yes the cg should have some protection to detect cases where we solve A*x = ε with ε small. I bet the other CG implementations might have this. I suppose you can get some ideas there.

If it turns out to be too tricky, then let's just leave the quick fix you propose.